### ISSUE TITLE
missing dpg-windows-GPU.yaml 

### ISSUE THREAD

**Issue by emorgand** [paninski-lab/deepgraphpose#11]
Hello, 

I am trying to install deepgraphpose on a windows operating system. I noticed that there is no dpg-windows-GPU.yaml within the src folder. I also noticed that there is tread within issues discussing use of deepgraphpose on windows. Where would I find the dpg-windows-GPU.yaml for installation?

Thanks for your help!

**Comment by waq1129** [comment]
Hi 

Have you installed DLC on windows successfully? You could do
conda env create -f dlc-windows-GPU.yaml

then follow the instruction.

Best

**Comment by emorgand** [comment]
> Hi
> 
> Have you installed DLC on windows successfully? You could do
> conda env create -f dlc-windows-GPU.yaml
> 
> then follow the instruction.
> 
> Best

This worked. Thanks again!

**Comment by reflexchem** [comment]
name: dgp
channels:
  - defaults
dependencies:
  - anyio=3.5.0=py37haa95532_0
  - argon2-cffi=21.3.0=pyhd3eb1b0_0
  - argon2-cffi-bindings=21.2.0=py37h2bbff1b_0
  - attrs=22.1.0=py37haa95532_0
  - backcall=0.2.0=pyhd3eb1b0_0
  - beautifulsoup4=4.11.1=py37haa95532_0
  - blas=1.0=mkl
  - bleach=4.1.0=pyhd3eb1b0_0
  - blosc=1.21.3=h6c2663c_0
  - bzip2=1.0.8=h2bbff1b_6
  - ca-certificates=2024.12.31=haa95532_0
  - certifi=2022.12.7=py37haa95532_0
  - cffi=1.15.1=py37h2bbff1b_3
  - colorama=0.4.6=py37haa95532_0
  - debugpy=1.5.1=py37hd77b12b_0
  - defusedxml=0.7.1=pyhd3eb1b0_0
  - entrypoints=0.4=py37haa95532_0
  - flit-core=3.6.0=pyhd3eb1b0_0
  - geos=3.8.0=h33f27b4_0
  - hdf5=1.14.5=ha36df97_2
  - icc_rt=2022.1.0=h6049295_2
  - idna=3.4=py37haa95532_0
  - importlib-metadata=4.11.3=py37haa95532_0
  - importlib_metadata=4.11.3=hd3eb1b0_0
  - importlib_resources=5.2.0=pyhd3eb1b0_1
  - ipykernel=6.15.2=py37haa95532_0
  - ipython=7.31.1=py37haa95532_1
  - ipython_genutils=0.2.0=pyhd3eb1b0_1
  - jedi=0.18.1=py37haa95532_1
  - jinja2=3.1.2=py37haa95532_0
  - jsonschema=4.17.3=py37haa95532_0
  - jupyter_client=7.4.9=py37haa95532_0
  - jupyter_core=4.11.2=py37haa95532_0
  - jupyter_server=1.23.4=py37haa95532_0
  - jupyterlab_pygments=0.1.2=py_0
  - libiconv=1.16=h2bbff1b_3
  - libsodium=1.0.18=h62dcd97_0
  - libxml2=2.9.14=h0ad7f3c_0
  - libxslt=1.1.35=h2bbff1b_0
  - lxml=4.9.1=py37h1985fb9_0
  - lz4-c=1.9.4=h2bbff1b_1
  - lzo=2.10=he774522_2
  - markupsafe=2.1.1=py37h2bbff1b_0
  - matplotlib-inline=0.1.6=py37haa95532_0
  - mistune=0.8.4=py37hfa6e2cd_1001
  - mkl=2020.2=256
  - mkl-service=2.3.0=py37h196d8e1_0
  - mkl_fft=1.2.0=py37h45dec08_0
  - mkl_random=1.1.1=py37h47e9c7a_0
  - mock=4.0.3=pyhd3eb1b0_0
  - nb_conda=2.2.1=py37_0
  - nb_conda_kernels=2.3.1=py37haa95532_0
  - nbclassic=0.5.2=py37haa95532_0
  - nbclient=0.5.13=py37haa95532_0
  - nbconvert=6.5.4=py37haa95532_0
  - nbformat=5.7.0=py37haa95532_0
  - nest-asyncio=1.5.6=py37haa95532_0
  - notebook=6.5.2=py37haa95532_0
  - notebook-shim=0.2.2=py37haa95532_0
  - numexpr=2.7.1=py37h25d0782_0
  - openssl=3.0.15=h827c3e9_0
  - packaging=22.0=py37haa95532_0
  - pandocfilters=1.5.0=pyhd3eb1b0_0
  - parso=0.8.3=pyhd3eb1b0_0
  - pickleshare=0.7.5=pyhd3eb1b0_1003
  - pip=22.3.1=py37haa95532_0
  - pkgutil-resolve-name=1.3.10=py37haa95532_0
  - prometheus_client=0.14.1=py37haa95532_0
  - prompt-toolkit=3.0.36=py37haa95532_0
  - psutil=5.9.0=py37h2bbff1b_0
  - pycparser=2.21=pyhd3eb1b0_0
  - pygments=2.11.2=pyhd3eb1b0_0
  - pyrsistent=0.18.0=py37h196d8e1_0
  - python=3.7.1=h33f27b4_4
  - python-dateutil=2.8.2=pyhd3eb1b0_0
  - python-fastjsonschema=2.16.2=py37haa95532_0
  - pywin32=305=py37h2bbff1b_0
  - pywinpty=2.0.10=py37h5da7b33_0
  - pyzmq=23.2.0=py37hd77b12b_0
  - send2trash=1.8.0=pyhd3eb1b0_1
  - setuptools=65.6.3=py37haa95532_0
  - shapely=1.7.1=py37h210f175_0
  - six=1.16.0=pyhd3eb1b0_1
  - sniffio=1.2.0=py37haa95532_1
  - soupsieve=2.3.2.post1=py37haa95532_0
  - sqlite=3.45.3=h2bbff1b_0
  - terminado=0.17.1=py37haa95532_0
  - tinycss2=1.2.1=py37haa95532_0
  - tornado=6.2=py37h2bbff1b_0
  - traitlets=5.7.1=py37haa95532_0
  - typing-extensions=4.4.0=py37haa95532_0
  - typing_extensions=4.4.0=py37haa95532_0
  - vc=14.42=haa95532_4
  - vs2015_runtime=14.42.34433=he0abc0d_4
  - wcwidth=0.2.5=pyhd3eb1b0_0
  - webencodings=0.5.1=py37_1
  - websocket-client=0.58.0=py37haa95532_4
  - wheel=0.38.4=py37haa95532_0
  - wincertstore=0.2=py37haa95532_2
  - winpty=0.4.3=4
  - wxpython=4.0.4=py37ha925a31_0
  - xz=5.6.4=h4754444_1
  - zeromq=4.3.4=hd77b12b_0
  - zipp=3.11.0=py37haa95532_0
  - zlib=1.2.13=h8cc25b3_1
  - zstd=1.5.6=h8880b57_0
  - pip:
      - absl-py==2.1.0
      - astor==0.8.1
      - chardet==5.2.0
      - charset-normalizer==3.4.1
      - click==8.1.8
      - cycler==0.11.0
      - decorator==4.4.2
      - dlclibrary==0.0.7
      - easydict==1.13
      - filelock==3.12.2
      - filterpy==1.4.5
      - fonttools==4.38.0
      - fsspec==2023.1.0
      - gast==0.2.2
      - google-pasta==0.2.0
      - grpcio==1.62.3
      - h5py==2.10.0
      - huggingface-hub==0.16.4
      - imageio==2.31.2
      - imageio-ffmpeg==0.5.1
      - imgaug==0.4.0
      - intel-cmplr-lib-ur==2025.0.5
      - intel-openmp==2025.0.5
      - joblib==1.3.2
      - keras-applications==1.0.8
      - keras-preprocessing==1.1.2
      - kiwisolver==1.4.5
      - llvmlite==0.39.1
      - markdown==3.4.4
      - matplotlib==3.0.3
      - moviepy==1.0.3
      - msgpack==1.0.5
      - msgpack-numpy==0.4.8
      - networkx==2.6.3
      - numba==0.56.4
      - numpy==1.21.6
      - opencv-python==3.4.5.20
      - opt-einsum==3.3.0
      - pandas==1.3.5
      - patsy==1.0.1
      - pillow==9.5.0
      - proglog==0.1.10
      - protobuf==3.20.3
      - pyparsing==3.1.4
      - pytz==2025.1
      - pywavelets==1.3.0
      - pyyaml==6.0.1
      - requests==2.31.0
      - ruamel-yaml==0.18.10
      - ruamel-yaml-clib==0.2.8
      - scikit-image==0.16.2
      - scikit-learn==1.0.2
      - scipy==1.7.3
      - statsmodels==0.10.1
      - tables==3.7.0
      - tabulate==0.9.0
      - tcmlib==1.2.0
      - tensorboard==1.15.0
      - tensorflow==1.15.0
      - tensorflow-estimator==1.15.1
      - tensorpack==0.11
      - termcolor==2.3.0
      - tf-slim==1.1.0
      - threadpoolctl==3.1.0
      - tifffile==2021.11.2
      - torch==1.12.0
      - tqdm==4.67.1
      - umf==0.9.1
      - urllib3==1.26.15
      - werkzeug==2.2.3
      - wrapt==1.16.0

**Comment by reflexchem** [comment]
not sure If this answers the issue. Created a .yaml to run DGP on CPU. python=3.7.1
